docs/content/en/4.providers/glide.md

```diff
@@ -2,10 +2,10 @@
 title: Glide Provider
 description: 'Nuxt Image has first class integration with Glide'
 navigation:
-  title: glide
+  title: Glide
```

Comment:
I capitalised the Glide title in the docs since it was annoying me that I made it lowercase.